linux_os/guide/system/accounts/accounts-banners/gui_login_banner/dconf_gnome_banner_enabled/ansible/ubuntu2004.yml

```diff
@@ -0,0 +1,29 @@
+# platform = multi_platform_ubuntu
+# reboot = false
+# strategy = unknown
+# complexity = low
+# disruption = medium
+- name: "Uncomment banner-message-enable for Login Warning Banner"
+  lineinfile:
+    path: /etc/gdm3/greeter.dconf-defaults
+    regexp: ^(#.*)(banner-message-enable=true)
```

Comment:
Remove the `true` and instead use:
`^(#.*)(banner-message-enable=)`

Which means we can ignore if set to false since the next task will correct the option, or create if a commented out option does not exist.